add gitignore, readme and additional file structure (#1)
makerdao/executive-votes: README.md

# executive-votes

This repository stores current and historical executive votes in Sky Protocol

## Directory Structure

The `executive-votes` repository is organized as follows:

```
executive-votes/
├── active/ # Contains executive votes currently under consideration
│ └── executive-vote-active-proposal.md
├── 2025/ # Contains executive votes posted in 2025
│ └── e.g. executive-vote-2025-01-15-update-oracle.md
├── ... # Additional year directories as needed
├── templates/ # Contains templates for creating new executive vote files
│ └── executive-vote-template.md
├── index.json # (Optional) An index of all executive votes for quick lookup
└── README.md # This file
```

## File Naming Convention

Executive vote files should follow the pattern: `executive-vote-<yyyy-mm-dd>-<description>.md`.

- `<yyyy-mm-dd>`: The date the vote was proposed.
- `<description>`: A brief, hyphenated description of the vote's purpose (e.g., `update-oracle`, `parameter-change`).

## Components

> - **`active/`**: Holds Markdown files for executive votes that are currently live or awaiting execution.

Author: ```suggestion
- **`active/`**: Holds a proposals.json file that lists executive votes that are currently live or awaiting execution.
```
